src/query/vacuum.c

```diff
@@ -6868,7 +6872,12 @@ vacuum_copy_log_page (THREAD_ENTRY * thread_p, LOG_PAGEID log_pageid, BLOCK_LOG_
   else
 #endif /* SERVER_MODE */
     {
-      if (logpb_fetch_page (thread_p, log_pageid, log_page_p) == NULL)
+      LOG_LSA req_lsa;
+
+      req_lsa.pageid = log_pageid;
+      req_lsa.offset = DB_PAGESIZE;
```

Comment:
@razvanradulescu 

`DB_PAGESIZE` should be `LOG_PAGESIZE`, since the size of a log page may be different from that of db page.